### PR TITLE
Use dor-services-app endpoint for administrative tags

### PR DESCRIPTION
**Pull request by mjgiarlo**
Connects to sul-dlss/dor-services-app#679

Includes:
* Bump to latest dor-services-client
* Remove outdated docker-compose cruft from Travis CI configuration
* Correct old fixtures now that we have stricter APIs:
  * Rename `ti057...` druids to `tj057...` since `i` is not a valid druid character
  * Add descMetadata to `pb873...` object, else it no longer has its title indexed due to semi-recent changes to indexing
* Stop using invalid druids (*e.g.*, `oo201...`) in tests

## Why was this change made?

To support SDR evolution goals around administrative tags.

## Was the documentation (README, DevOpsDocs, wiki, consul, etc.) updated?

No.

## Does this change affect how this application integrates with other services?

Yes. Tested locally by spinning up affected services in containers and getting the specs to pass. Also: https://github.com/sul-dlss/infrastructure-integration-test/pull/34

### CODE REVIEW

**Comment by mjgiarlo** [comment]
I believe the builds are breaking because with this change, Argo needs related changes in the dor-services-app and dor-indexing-app images. The tests pass locally, where I have those updated images. Once the related dor-services-app and dor_indexing_app PRs are merged, I'll restart this build.

File: .travis.yml [hunk 1]
```diff
@@ -14,12 +14,6 @@ email: false
 before_install:
   # Get bundler 2.0 for ruby 2.6.4
   - gem install bundler
-  # Travis's docker-compose is too old to handle v3.6 docker-compose templates.
```
Comment (jcoyne) [inline .travis.yml]: 👏

File: app/views/items/_tags_ui.html.erb [hunk 2]
```diff
@@ -1,27 +1,27 @@
-<%=form_tag url_for(:controller => :items, :action => :tags, :id => @object.pid, :update => 'true') do%>
-  <%@object.tags.each_with_index do |tag, count|%>
+<%= form_tag url_for(:controller => :items, :action => :tags, :id => @pid, :update => 'true') do %>
+  <% @tags.each_with_index do |tag, count| %>
     <div class="form-group">
       <div class="input-group">
-        <input class="form-control tag" name="tag<%=count + 1%>" id="tag<%=count + 1%>" type="text" value="<%=tag%>"/>
+        <input class="form-control tag" name="tag<%= count + 1 %>" id="tag<%= count + 1 %>" type="text" value="<%= tag %>"/>
```
[the remark this answers is not in the document]
Comment (jcoyne) [reply app/views/items/_tags_ui.html.erb]: Can you remove the trailing `/` here?

[the remark this answers is not in the document]
Comment (mjgiarlo) [reply app/views/items/_tags_ui.html.erb]: @jcoyne I considered it, but Argo pretty consistently uses `<input ... />`, and I was reluctant to introduce inconsistency here for little gain other than silencing hound. OK to skip for now?

[the remark this answers is not in the document]
Comment (jcoyne) [reply app/views/items/_tags_ui.html.erb]: Well, it's the html5 standard. It's not just hound.

**Pull request merged**